manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/provider/CustomSnippetsProvider.java

```diff
@@ -51,9 +52,11 @@
     CustomSnippetsProcessor customSnippetsProcessor;
 
     public void init(ServiceSpec serviceSpec) throws HyscaleException {
-        TypeReference<List<String>> listTypeReference = new TypeReference<List<String>>() {};
-        List<String> k8sSnippetFilePaths= serviceSpec.get(HyscaleSpecFields.k8sPatches,listTypeReference);
-        this.kindVsCustomSnippets = customSnippetsProcessor.processCustomSnippetFiles(k8sSnippetFilePaths);
+        if(serviceSpec != null){
```

Comment:
Keep an inverse check
if service spec is null just return
The main logic can be kept afterwards